app/server-routes/src/main/scala/org/bitcoins/server/routes/BitcoinSRunner.scala

```diff
@@ -118,4 +120,34 @@ trait BitcoinSRunner extends BitcoinSLogger {
       sys.exit(1)
     }(scala.concurrent.ExecutionContext.Implicits.global)
   }
+
+  def getUserPasswords: (Option[AesPassword], Option[String]) = {
+    val kmConf = KeyManagerAppConfig(datadir, baseConfig)
+    kmConf.start()
+
+    if (!kmConf.seedExists()) {
+      logger.info("Creating new wallet!")
+      // Print to user if their log level is too high
+      if (!logger.isInfoEnabled) {
+        println("Creating new wallet!")
+      }
+      print("Enter wallet password (empty for unencrypted wallet): ")
+      val aesPassStr: String = System.console().readPassword().mkString
+      val aesPasswordOpt = AesPassword.fromStringOpt(aesPassStr)
+
+      print("Confirm wallet password: ")
+      val confirmAesStr: String = System.console().readPassword().mkString
+      require(aesPassStr == confirmAesStr, "Wallet passwords did not match")
+
```

Comment:
So if I understand correctly, the second time the users fires up `appServer` they MUST have set the configs correctly in `bitcoin-s.conf`?

If that is the case, we should definitely link to resources for how to set these options. Basically link to the bitcoin-s.org configuration example IMO.

Reply:
Maybe we should add it to the their conf manually?

Reply:
Modifying a users configuration file is probably not a good idea. It will lead to confusion in my opinion.